cves/CVE-2018-14574.yml

```diff
@@ -298,7 +320,17 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I would say that this is a mistake in the Requirements of the project. This
```

Comment:
I don't understand what you mean by this being a mistake in the requirements. I get that this was introduced by an optimization that was not properly reviewed, but I don't see how that is caused by the requirements. Maybe explain a bit more what you mean by this?